CHANGES.md

```diff
@@ -8,6 +8,10 @@
 
 ### Fixed
 
+- Fix project name detection from `dune-project`. The parser could get confused
+  when opam file generation is used. Now it only considers `(name X)` at the
+  beginning of a line. (#445, @emillon)
```

Comment:
I was a bit confused by this because I expected to see a `^`-equivalent in the regex, but I think what it means to say is "on its own line"?

Reply:
Ah right I forgot to add `bol`. Well, "first match" might be a better heuristic anyway. Should I document it as such?

Reply:
done!